spring-cassandra/src/main/java/org/baeldung/springcassandra/model/Person.java

```diff
@@ -0,0 +1,31 @@
+package org.baeldung.springcassandra.model;
+
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.data.cassandra.core.mapping.PrimaryKey;
+import org.springframework.data.cassandra.core.mapping.Table;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+@Builder
+@Table
```

Comment:
We're relying on these annotations producing `equals`. I presume that `Data` does, but it would be simpler to just HAVE an equals.